certificate/certificate.go

```diff
@@ -301,6 +321,17 @@ func getExtKeyUsages(cert *x509.Certificate) []string {
 	return usage
 }
 
+func getExtKeyUsageOIDs(cert *x509.Certificate) []string {
+	usage := make([]string, 0)
```

Comment:
If you're initializing this with `make`, set the initial size to `len(cert.ExtKeyUsage) + len(cert.UnknownExtKeyUsage)`, or do what @jvehent suggests.